jax/_src/nn/functions.py

```diff
@@ -353,9 +357,11 @@ def glu(x: Array, axis: int = -1) -> Array:
   See also:
     :func:`sigmoid`
   """
-  size = x.shape[axis]
```

Comment:
you can always do `np.shape(x)[axis]` if you don't want to explicitly cast to array

Reply:
Ack. I decided to keep the original version and convert `x` to an array instead.